libs/@local/hash-isomorphic-utils/src/generate-system-types.ts

```diff
@@ -65,6 +72,36 @@ const generateTypes = async (
   console.log(`Done generating ${label} types.`);
 };
 
+const replaceInFile = async (
+  filePath: string,
+  search: RegExp,
+  replace: string,
+) => {
+  const contents = await fs.readFile(filePath, "utf-8");
+  const result = contents.replace(search, replace);
+
+  await fs.writeFile(filePath, result, "utf-8");
+};
+
+const replaceInDirectory = async (
```

Comment:
The system types need regenerating for this to take effect – I've created H-2452 to do this next week, because there'll also be some opportunities to remove type hackery we were doing before and I don't want to pile more stuff onto this PR